packages/eslint-config/eslint-config.json

```diff
@@ -14,17 +14,59 @@
     "commonjs": false
   },
   "globals": {
-    "BigInt": "readonly",
     "assert": "readonly",
     "console": "readonly",
-    "harden": "readonly",
-    "lockdown": "readonly",
     "Compartment": "readonly",
     "StaticModuleRecord": "readonly",
     "TextDecoder": "readonly",
     "TextEncoder": "readonly",
     "URL": "readonly",
-    "URLSearchParams": "readonly"
+    "URLSearchParams": "readonly",
+
+    // Allow what SES makes powerless, copied from its whitelist
+    // *** Constructor Properties of the Global Object
```

Comment:
JS comments in a JSON file? If this is read by something other than a JSON parser, note that at the top, maybe?